.changeset/tasty-trainers-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'wingman-fe': patch
+---
+
+**MockSalaryDetails:** Uncamelcase Storybook group name
```

Comment:
Does this need a changeset? It only affects Storybook right, not the npm package?

Reply:
I've gone from always forgetting the changeset to creating them on autopilot 🙃